cdap-docs/admin-manual/source/troubleshooting.rst

```diff
@@ -2,13 +2,29 @@
     :author: Cask Data, Inc.
     :copyright: Copyright © 2014 Cask Data, Inc.
 
-============================================
-Troubleshooting CDAP
-============================================
+.. _admin-troubleshooting:
```

Comment:
Good question. In the current (3.2.1) documentation, we have three "Troubleshooting" sections:

- Developers' Manual: Testing and Debugging: Troubleshooting
  (http://docs.cask.co/cdap/3.2.1/en/developers-manual/testing/troubleshooting.html)
- CDAP Administration Manual: Operations: Troubleshooting
  (http://docs.cask.co/cdap/3.2.1/en/admin-manual/operations/troubleshooting.html)
- FAQs
  (http://docs.cask.co/cdap/3.2.1/en/faqs/index.html)

The Admin one in the middle of that list is the same as the one in the PR comment above. I would suggest that they be consolidated into one, and so in that case, remove this document and link to the FAQs instead.